### ISSUE TITLE
Setting to follow to running cells in a notebook

### ISSUE THREAD

**Issue by misolori** [microsoft/vscode#107919]
When running all cells in a notebook that has large cells, sometimes it's hard to follow along where the current cell is especially when the execution happens fast. I would love for an option that "follows" the actively running cell as it moves through the notebook. Right now I have to manually scroll or use the keyboard to follow.

![ezgif com-gif-maker (6)](https://user-images.githubusercontent.com/35271042/94852271-7a098780-03de-11eb-9d97-949001dd906a.gif)

**Comment by rebornix** [comment]
A mode for "auto reveal cell in execution"
